src/menu.c

```diff
@@ -119,6 +119,12 @@ void open_browser(const char *in)
                             string_append(settings.browser, url, " ");
                         char **cmd = g_strsplit(browser_cmd, " ", 0);
                         execvp(cmd[0], cmd);
+                        // execvp won't return if it's successful
+                        // so, if we're here, it's definitely an error
+                        fprintf(stderr, "Warning: failed to execute '%s': %s\n",
+                                        settings.browser,
```

Comment:
`settings.browser` is being overwritten above by the `string_append` call and can potentially be empty or point to junk here. Perhaps printing `browser_cmd` is better.

Reply:
Albeit not outdated, I've handled this in the `string_append` lines above.